.github/workflows/certification.yml

```diff
@@ -217,8 +220,12 @@ jobs:
 
         set +e
         gotestsum --jsonfile ${{ env.TEST_OUTPUT_FILE_PREFIX }}_certification.json \
-          --junitfile ${{ env.TEST_OUTPUT_FILE_PREFIX }}_certification.xml --format standard-quiet -- \
-          -count=1 -timeout=15m
+          --junitfile ${{ env.TEST_OUTPUT_FILE_PREFIX }}_certification.xml --format standard-verbose -- \
```

Comment:
Is changing the --format required for generating the coverage file?

Reply:
standard-quiet is fine for coverage. Changing back.